modules/experimental/pages/robust.adoc

```diff
@@ -0,0 +1,102 @@
+= Robust
+
+`Unit ability`
+
+An experimental rule to allow some normal units to have Damage Capacity `2`.
+
+This is similar to how Damage Capacity works with War Engine units.
+That said...
+
+IMPORTANT: Units that have the {robust} ability are not War Engine units — do not apply the War Engine rules to them.
+
+In brief, it takes two hits to destroy a {robust} unit, and each {robust} unit counts as two normal units for both hit allocation and detachment strength.
+
+[NOTE]
+====
+We're testing this ability as a way to make some units tougher than the standard rules allow, but not as tough as War Engine units.
+Candidates for this ability include Space Marine Land Raiders, Ork Stompas, and Chaos Greater Daemons.
+====
+
+---
+
+Where your unit has the {robust} ability:
+
+* The unit starts the game with Damage Capacity `2`.
+* When you calculate the detachment strength, count the unit as equal to its remaining Damage Capacity.
+* Each hit reduces its Damage Capacity by one point.
+* If the unit's Damage Capacity is reduced to 0 then it is destroyed.
+
+Where you attack a detachment that includes units that have the {robust} ability:
+
+* Each {robust} unit counts as *two* units for hit allocation.
+This means that when you allocate hit dice and you reach a {robust} unit, you must allocate *two* dice to that unit before you allocate dice to more distant targets.
+// TODO: Do we still have to allocate two dice even if the unit has only 1 DC remaining?
+* Where possible, you must allocate two dice that score equal to or higher than the {robust} unit's Armour value, *or* two dice that score lower.
```

Comment:
New suggestion, per earlier thread. Based on yours, but with some tweaks to wording. I also added a little explanation of the last bullet in parentheses at the end — but I'm not wedded to it if you're not keen on that addition.

```suggestion
* Each {robust} unit counts as either *one* or *two* units for hit allocation, according to the number of Damage Capacity points it has left.
This means that when you allocate hit dice and you reach a {robust} unit, you must allocate two dice if it is undamaged or one die if it is damaged.
Then continue to allocate dice to more distant targets in the usual way.
* Where possible, all dice that you allocate to a {robust} unit must score equal to or higher than its Armour value, *or* all dice that you allocate must score lower.
(To minimise the number of damaged units that you need to track in the game.)
```

Reply:
Oops, didn't need that parenthetical bit as we've covered it in the note in the next line, which is probably a better way to do it anyway :-D

```suggestion
* Each {robust} unit counts as either *one* or *two* units for hit allocation, according to the number of Damage Capacity points it has left.
This means that when you allocate hit dice and you reach a {robust} unit, you must allocate two dice if it is undamaged or one die if it is damaged.
Then continue to allocate dice to more distant targets in the usual way.
* Where possible, all dice that you allocate to a {robust} unit must score equal to or higher than its Armour value, *or* all dice that you allocate must score lower.
```

Reply:
Looking good.